pyproject.toml

```diff
@@ -21,6 +21,120 @@ packages = [{ include = "langflow", from = "src/backend" }]
 include = ["src/backend/langflow/*", "src/backend/langflow/**/*"]
 documentation = "https://docs.langflow.org"
 
+[tool.uv.sources]
+langflow-base = {path = "src/backend/base"}
+
+[tool.hatch.build.targets.wheel]
+packages = ["src/backend/langflow"]
+
+[project]
+name = "langflow"
+version = "1.0.18"
+description = "A Python package with a built-in web application"
+requires-python = ">=3.10,<3.13"
+# Define your main dependencies here
+dependencies = [
+    "langflow-base==0.0.96",  # Local editable package
```

Comment:
How does this work if I run `python -m langflow run` in my repo - does it use the pinned 0.0.96 version from pypi or the path from the `tool.uv.sources` above?

Reply:
Good eyes, I *think* we want:
    "langflow-base @ file://./src/backend/base",  # Local editable package

which is what I had originally in here but I don't know if we have a good way of keeping it from getting accidentally committed. uv seems to replace it with the 0.0.96 base automatically when we build.

Reply:
This was fixed. It has to be the version but then add it as a workspace package